library/lib-discovery-client-starter/src/main/java/com/sokima/lib/discoveryclient/starter/config/DiscoveryClientAutoConfiguration.java

```diff
@@ -1,11 +1,14 @@
 package com.sokima.lib.discoveryclient.starter.config;
 
-import org.springframework.boot.autoconfigure.AutoConfiguration;
 import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 import org.springframework.context.annotation.PropertySource;
 
+@Configuration
```

Comment:
Can we leave `@Autoconfiguration`?
```suggestion
@Configuration
```

Reply:
We can.